src/DTS/Render/RenderHelper.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+
+namespace HomeCEU\DTS\Render;
+
+
+class RenderHelper {
+  public static function extractExpectedPartialsFromTemplate(string $template): array  {
+    preg_match_all('/{{>([^\}}]+)}}/', $template, $matches);
```

Comment:
one of the few places you should for sure use a comment is when you have regex 

I often link to a regex101 page which demo's the regex and what it does and captures etc....

Reply:
This code should actually be deleted, unless we want to use it later. Which we might. 

the regex101 link is a good idea